src/ScoutingMain.elm

```diff
@@ -181,13 +187,14 @@ subscriptions =
         , Sub.map TeamDataMsg <| TeamData.subscriptions
         , Sub.map TeleopDataMsg <| Teleop.subscriptions
         , Sub.map ClimbingDataMsg <| Climbing.subscriptions
+        , BE.onResize (\w h -> ScreenSize <| Element.classifyDevice <| DeviceSize w h)
```

Comment:
I believe that a simple `BE.onResize << ScreenSize << Element.classifyDevice << DeviceSize` would work the same way.